src/test/test_util.c

```diff
@@ -6286,6 +6286,10 @@ test_util_map_anon_nofork(void *arg)
 #define UTIL_TEST_NO_WIN(n, f) { #n, NULL, TT_SKIP, NULL, NULL }
 #define UTIL_TEST_WIN_ONLY(n, f) UTIL_TEST(n, (f))
 #define UTIL_LEGACY_NO_WIN(n) UTIL_TEST_NO_WIN(n, 0)
+#elif defined(__ANDROID__)
+#define UTIL_TEST_NO_WIN(n, f) { #n, NULL, TT_SKIP, NULL, NULL }
```

Comment:
WIN should only mean "Windows", not "Android".  If we're going to change the meaning of these tests, we should also change their name.

Reply:
Ok, what do you want to call it?  I didn't rename it because I don't really understand what its filtering out.

Reply:
I removed this part from this pull request.  I'll leave that to someone who knows that stuff.  For now, the pwdb test fails on Android.